internal/api/server.go

```diff
@@ -162,7 +162,7 @@ func (s *Server) setupRouter() (*gin.Engine, error) {
 
 	r.POST("/init", s.registerInitServerHandler())
 
-	requireProdMode := s.requireServerMode(model.ModeProd)
+	requireProdMode := s.requireServerMode(model.ModeEnterprise)
```

Comment:
we should also rename `requireProdMode` to `requireEnterpriseMode` to avoid confusion in the future.
Same applies to any other variables using prod